packages/core/i18n/src/i18n.ts

```diff
@@ -32,6 +32,43 @@ export const createI18n = <MessageSource extends Record<string, any>>
     intlCache,
   )
 
+  /**
+   * Formats a unix timestamp into a formatted date string
+   * @param {Number} timestamp a unix timestamp in seconds
+   * @returns a date string formatted like 'Apr 6, 2022 10:50'
+   */
+  const formatUnixTimeStamp = (timestamp: number): string => {
```

Comment:
you might want to move those functions below line 76, otherwise it's not replay cleat where `intl.` coming from...

Reply:
Done